recipe/meta.yaml

```
@@ -11,19 +11,18 @@ source:
   folder: HiggsAnalysis/CombinedLimit
   patches:
     - clang_warnings.patch
-    - makefile.patch
 
 build:
   number: 1
   skip: true  # [win]
 
 requirements:
   build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
     - {{ stdlib('c') }}
     - {{ compiler('cxx') }}
-    - make
-    - python  # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}  # [build_platform != target_platform]
+    - cmake
```

```suggestion
    - cmake
    - root_base  # [build_platform != target_platform]
```

If you're getting `Bad CPU type in executable` then you might need to have your `host` requirements get added as `build` requirements for cross-compile (same reason as for the `python` stuff above).

What's the full error?

---

that's all there is, but I think you're right about putting `root_base` in build requirements. We'll see soon